docs/en/connector-v2/Config-Encryption-Decryption.md

```diff
@@ -0,0 +1,189 @@
+
+## How to use
+
+SeaTunnel comes with the function of base64 encryption and decryption, but it is not recommended for production use, it is recommended that users implement custom encryption and decryption logic. You can refer to this chapter [How to implement user-defined encryption and decryption](#How to implement user-defined encryption and decryption) get more details about it.
+
+Next, I'll show how to quickly use SeaTunnel's own `base64` encryption:
+
+1. And a new option `shade.identifier` in env block of config file, this option indicate what the encryption method that you want to use, in this example, we should add `shade.identifier = base64` in config as the following shown:
+
+   ```hocon
+   #
+   # Licensed to the Apache Software Foundation (ASF) under one or more
+   # contributor license agreements.  See the NOTICE file distributed with
+   # this work for additional information regarding copyright ownership.
+   # The ASF licenses this file to You under the Apache License, Version 2.0
+   # (the "License"); you may not use this file except in compliance with
+   # the License.  You may obtain a copy of the License at
+   #
+   #     http://www.apache.org/licenses/LICENSE-2.0
+   #
+   # Unless required by applicable law or agreed to in writing, software
+   # distributed under the License is distributed on an "AS IS" BASIS,
+   # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   # See the License for the specific language governing permissions and
+   # limitations under the License.
+   #
+
+   env {
+     execution.parallelism = 1
+     shade.identifier = "base64"
+   }
+
+   source {
+     MySQL-CDC {
+       schema {
+         fields {
+           name = string
+           age = int
+           sex = boolean
+         }
+       }
+       result_table_name = "fake"
+       parallelism = 1
+       server-id = 5656
+       port = 56725
+       hostname = "127.0.0.1"
+       username = "seatunnel"
+       password = "seatunnel_password"
+       database-name = "inventory_vwyw0n"
+       table-name = "products"
+       base-url = "jdbc:mysql://localhost:56725"
+     }
+   }
+
+   transform {
+   }
+
+   sink {
+     # choose stdout output plugin to output data to console
+     Clickhouse {
+       host = "localhost:8123"
+       database = "default"
+       table = "fake_all"
+       username = "seatunnel"
+       password = "seatunnel_password"
+
+       # cdc options
+       primary_key = "id"
+       support_upsert = true
+     }
+   }
+   ```
+2. Using the shell based on different calculate engine to encrypt config file, in this example we use zeta:
+
+   ```shell
+   ${SEATUNNEL_HOME}/bin/seatunnel.sh --config config/v2.batch.template --encrypt
+   ```
+
+   Then you can see the encrypted configuration file in the terminal:
+
+   ```log
+   2023-02-20 17:50:58,319 INFO  org.apache.seatunnel.core.starter.command.ConfEncryptCommand - Encrypt config: 
+   {
```

Comment:
Output original file format?

example
```json
env {
}

source {

}

transform {

}

sink {

}
```

Reply:
This is limited by the format interface provided by typesafe config, this requirement I had concerned but not support this format.